Source/Core/CommandLineOptions.cs

```diff
@@ -2452,8 +2453,17 @@ be n (default 500)
                 aggressive and it may create an exponential number of splits.
 
   /randomSeed:<n>
-                Randomise arbitrary decisions during verification condition generation, 
-                and during solving   
```

Comment:
I would keep a high-level explanation first, rather than starting with a note.  Maybe something like this:

> Turn on randomization in the input that Boogie passes to the SMT solver and in the SMT solver itself.  

… and then most of the text that you had below